pkgs/by-name/ne/nextcloud-client-bin/package.nix

```diff
@@ -0,0 +1,46 @@
+{ p7zip
+, libarchive
+, stdenvNoCC
+, lib
+, fetchurl
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "nextcloud-client-bin";
+  version = "3.13.0";
+  src = fetchurl {
+    url = "https://github.com/nextcloud-releases/desktop/releases/download/v${finalAttrs.version}/Nextcloud-${finalAttrs.version}.pkg";
+    hash = "sha256-xqrOJxkFCQcTN7N0ZEjT85zn+u/o/fJUMZdaMBizls0=";
+  };
+
+  unpackPhase =
+  ''
+    7z x $src
+    bsdtar -xf Payload~
+  '';
+
+  nativeBuildInputs = [
+    p7zip
+    libarchive
+  ];
+
+  installPhase =
+  ''
+    runHook preInstall
+
+    mkdir -p $out/Applications
+    ls
```

Comment:
```suggestion
```
We don't really need `ls` here

Reply:
Removed. That was a thing for testing that I forgot about